doc/api/process.md

```diff
@@ -2305,6 +2313,32 @@ import { loadEnvFile } from 'node:process';
 loadEnvFile();
 ```
 
+Override local environment variables using override option
+
+```cjs
+const { loadEnvFile } = require('node:process');
+loadEnvFile('.env', { override: true });
+```
+
+```mjs
+import { loadEnvFile } from 'node:process';
+loadEnvFile('.env', { override: true });
+```
+
+This API is available through the [`--env-file-override-local`][] flag.
+
+Load environment variables with options only
+
+```cjs
+const { loadEnvFile } = require('node:process');
+loadEnvFile({ override: true });
```

Comment:
Can you mention the behavior as well? Like assert.deepEqual(process.env.existing_value, newValue)